server/src/routers/router.es6

```diff
@@ -0,0 +1,28 @@
+'use strict';
```

Comment:
Should this just be index.js for the routers directory? or even possibly `/routers/specialuses`